src/__tests__/resetpassword/ForgotPassword.test.jsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import Enzyme, { shallow } from "enzyme";
+import moxios from "moxios";
+import Adapter from "enzyme-adapter-react-16";
+import configureMockStore from "redux-mock-store";
+import thunk from "redux-thunk";
+import { ForgotPassword } from "../../components/auth/ForgotPasswordComp";
+import { FORGOT_PWD } from "../../redux/action_types";
+import { forgotPassword } from "../../redux/actions/ForgotPasswordAction";
+
+Enzyme.configure({ adapter: new Adapter() });
+const middlewares = [thunk];
+const mockStore = configureMockStore(middlewares);
+//   configure Mock store
+const store = mockStore({ response: { message: "check your email" } });
+describe("user action", () => {
+  beforeEach(() => {
+    moxios.install();
+  });
+  afterEach(() => {
+    moxios.uninstall();
+  });
+  const email = "test@gmail.com";
+  it("should dispatch an action to generate email address", () => {
+    const expectedAction = [
+      {
+        type: FORGOT_PWD,
+        responseData: "test@gmail.com"
+      }
+    ];
+
+    store.dispatch(forgotPassword(email));
+    expect(store.getActions()).toEqual(expectedAction);
+  });
+});
+describe("forgotPassword component", () => {
+  const dispatch = jest.fn();
+  const props = { dispatch, message: {} };
```

Comment:
Props are defined at the beginning of this page. Right before the first `describe()`.

const props = {
   dispatch: jest.fn()
}

Reply:
And how are you declaring it here but defining it right above?

Reply:
I have moved them up. Let me know your take.